packages/blade/src/components/Button/BaseButton/BaseButton.tsx

```diff
@@ -207,6 +216,9 @@ const BaseButton = ({
       type={type}
       borderWidth={borderWidth}
       borderRadius={borderRadius}
+      motionDuration={motionDuration}
+      motionEasing={motionEasing}
+      theme={theme}
```

Comment:
why to pass theme explicitly? it will be available in the styled component as props right?